src/screens/SUE/SueListScreen.tsx

```diff
@@ -80,29 +80,51 @@ export const SueListScreen = ({ navigation, route }: Props) => {
   const initialQueryVariables = route.params?.queryVariables ?? {
     initial_start_date: '1900-01-01T00:00:00+01:00'
   };
+  const limit = 20;
   const [queryVariables, setQueryVariables] = useState(initialQueryVariables);
   const [refreshing, setRefreshing] = useState(false);
   const [isOpening, setIsOpening] = useState(true);
+  const [sorting, setSorting] = useState(SORT_BY.REQUESTED_DATE_TIME);
 
-  const { data, isLoading, refetch } = useQuery(
+  const { data, isLoading, refetch, fetchNextPage, hasNextPage } = useInfiniteQuery(
     [
       query,
       {
         ...queryVariables,
-        start_date: queryVariables.start_date || queryVariables.initial_start_date
+        limit,
+        offset: 0,
+        start_date: queryVariables.start_date || queryVariables.initial_start_date,
+        sort_attribute: sorting
       }
     ],
-    () =>
+    ({ pageParam = 0 }) =>
       getQuery(query)({
         ...queryVariables,
-        start_date: queryVariables.start_date || queryVariables.initial_start_date
-      })
+        limit,
+        offset: pageParam,
+        start_date: queryVariables.start_date || queryVariables.initial_start_date,
+        sort_attribute: sorting
+      }),
```

Comment:
please reduce duplication with setting

```
const [queryVariables, setQueryVariables] = useState({ 
  ...initialQueryVariables, limit,
  offset: 0,
  start_date: queryVariables.start_date || queryVariables.initial_start_date,
  sort_attribute: sorting 
});
```

and update just `offset: pageParam` down here

Reply:
I reduced the code repetition a bit, but the `sort_attribute` variable should remain as it is.